tools/razzle.cmd

```diff
@@ -53,7 +53,7 @@ if not defined MSBUILD (
     goto :EXIT
 )
 
-set PATH=%PATH%"%MSBUILD%\..";

```

Comment:
Interesting. This only works because `PATH` is set on line 13 to end with a `;`!
Also, thanks for fixing the quotes.